src/test/java/com/tabnine/binary/BinaryVersionFetcherTests.java

```diff
@@ -30,10 +30,10 @@ public class BinaryVersionFetcherTests {
   @InjectMocks private BinaryVersionFetcher binaryVersionFetcher;
 
   @BeforeEach
-  public void setUp() throws BackingStoreException {
+  public void setUp() throws BackingStoreException, NoValidBinaryToRunException {
     Preferences preferences = Preferences.userNodeForPackage(BootstrapperSupport.class);
     preferences.clear();
-    when(binaryRemoteSource.fetchPreferredVersion(StaticConfig.getTabNineBundleVersionUrl()))
+    when(binaryRemoteSource.fetchPreferredVersion(anyString()))
```

Comment:
its better if you have the specific string to compare against - why did you change this? what broke the test?
same question for all the `anyString` changes

Reply:
because now I need to mock it inside